### ISSUE TITLE
Current packaged version of Chromium adds whitespace to the rendered HTML, latest Chromium stable does not

### ISSUE THREAD

**Issue by GlennVin** [puppeteer/puppeteer#4595]
I'm using puppeteer to make PDFs because it's quite amazing at it. However, the default minimal margin is added to the top of the first page. I have margins set to 0 on everything in my CSS and in every browser I've tried, including chromium itself, the margins are respected by the PDF printer.

When I set custom margins in the PDF print view in both Chrome and Chromium, it also works fine. However, when I pass margins via Puppeteer, a default margin is still added only to the top of the very first page.

**Environment:**

Puppeteer version: v1.17.0
Platform / OS version: Windows 7 enterprise
Node.js version: v10.16.0

**What is the expected result?**
Page renders without the added whitespace on top like it does in Chrome and Chromium without puppeteer

**What happens instead?**
Packaged Chromium adds a whitespace on top

**code**

```
  await page.setContent(html);
  const pdf = await page.pdf({
        preferCSSPageSize: true,
        printBackground: true,
        margin: {
            top: 0, /* ignored only on the first page, the others work fine*/
            left: 0,
            right: 0,
            bottom: 0,
        }
    });
```
css print dimensions
```
    @page {
        size: 8.26in 11.69in;
        margin: 0;
    }
```
Since this shifts all positions down, it breaks every subsequent element. Am I messing up somewhere or is this a known problem?

EDIT:
After fiddling around with this issue more, it appears that the current packaged version of Chromium itself renders this whitespace on top as part of the page for some unknown reason. The latest stable version of Chromium downloaded on 18/6/2019 does not do this for the same html.

**Comment by aslushnikov** [comment]
@GlennVin we just released v1.18.0 with an updated Chromium - can you please check if it works as expected for you?

**Comment by GlennVin** [comment]
> @GlennVin we just released v1.18.0 with an updated Chromium - can you please check if it works as expected for you?

It works as expected now, thank you so much!
